slides/01-introduction.qmd

```diff
@@ -452,15 +292,15 @@ knitr::include_graphics("images/whole-game-select.svg")
 knitr::include_graphics("images/whole-game-final.svg")
```

Comment:
I think adding speaker notes to this slide borked the image. I think that is some quarto bug we've seen before
<img width="1552" alt="Screen Shot 2022-07-24 at 2 23 41 PM" src="https://user-images.githubusercontent.com/19150088/180660832-38749ce5-99ec-4bec-9e61-0420fa64677c.png">

Reply:
Removed the speaker notes